ui/packages/catalog-ui-search/src/main/webapp/component/search-form-list/search-form-list.js

```diff
@@ -71,13 +71,32 @@ const FilterPadding = styled.div`
 class SearchForms extends React.Component {
   constructor(props) {
     super(props)
+    this.model = new Backbone.Model({
+      currentQuery: props.model,
+      searchForms: SearchFormCollection.getCollection(),
+    })
     this.state = {
       filter: '',
     }
   }
+  changeView(selectedForm, currentQuery) {
+    const sharedAttributes = selectedForm.transformToQueryStructure()
+    currentQuery.set({
+      type: 'custom',
+      ...sharedAttributes,
+    })
+    if (currentQuery.get('type') === 'custom') {
```

Comment:
Isn't this always guaranteed to be true here since you set the `currentQuery`'s type to custom in L85?

Reply:
hmm it looks that way, I just moved this function over. I don't see any way that would be false though